WalletWasabi.Fluent/Views/Wallets/WalletView.axaml

```diff
@@ -48,44 +51,50 @@
       </StackPanel>
 
     </c:ContentArea.Title>
-    <DockPanel VerticalAlignment="Top">
-
+    <DockPanel Name="Panel" VerticalAlignment="Stretch" HorizontalAlignment="Stretch">
+      <i:Interaction.Behaviors>
+        <behaviors:BoundsObserverBehavior Bounds="{Binding #Panel.Bounds, Mode=OneWay}"
+                                          Width="{Binding WidthSource, Mode=TwoWay}"
```

Comment:
Why do we need this behavior? we could simply bind #Panel.Bounds.Width?

Reply:
We need to pass bounds.width from panel to view model property. You can not bind to view model property in xaml so this behavior serves as proxy.

Reply:
ok